### DISCUSSION TITLE
404 Error upon deployment

### DISCUSSION THREAD

**Discussion by Jengas2015** [vercel/community#4262] [Help]
### Summary

I'm trying to deploy a project [here](https://gratitude-journal-puce.vercel.app/) and I'm getting a 404 error. Could anyone help me figure out what I'm missing? Much thanks in advance!

### Example

https://gratitude-journal-puce.vercel.app/

### Steps to Reproduce

Entering the site will reproduce the problem.

**Comment by elrumordelaluz** [comment]
Same here, no chance to solve this. Any update?
